src/test/java/api/support/builders/FeeFineBuilder.java

```diff
@@ -0,0 +1,72 @@
+package api.support.builders;
+
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+import java.util.UUID;
+
+import io.vertx.core.json.JsonObject;
+
+public class FeeFineBuilder extends JsonBuilder implements Builder {
+  private UUID id;
+  private UUID ownerId;
+  private String feeFineType;
+  private Double defaultAmount;
+  private String chargeNoticeId;
+  private String actionNoticeId;
```

Comment:
Can these be final?

Reply:
They can, if I initialize all of them with null in the no-argument constructor.

Reply:
If the sensible defaults for the builder are all null, then that is what I think should be done

Reply:
Done.